playground/backend/containers/go/Dockerfile

```diff
@@ -58,17 +58,6 @@ ENV PROPERTY_PATH=/opt/playground/backend/properties.yaml
 ## Copy build result
 COPY src/configs /opt/playground/backend/configs/
```

Comment:
Why are "configs" embeded in a Dockerfile? Given the kubernetes context? Why not rely on a ConfigMap instead of coupling the binary with configuration?

Reply:
These "configs" define how to construct command lines to run the Beam examples - I don't see any reason why anybody might need to change them without rebuilding the application and/or container